tests/foreman/api/test_contentview.py

```diff
@@ -410,6 +410,22 @@ def test_positive_restart_publish_via_dynflow(self):
 class ContentViewCreateTestCase(APITestCase):
     """Create tests for content views."""
 
+    def _apply_pacakge_filters(self, content_view, repo, package, inclusion=True):
```

Comment:
can move this one to part of API utili and make it public function ?

Reply:
Currently, this is used in a single place in the framework. Do you think we have to do this PR ? or we can do it in another PR as part of refactoring.